bcl2fastq/index_summary.py

```diff
@@ -40,31 +49,43 @@
                      "or unknown. This is due to multiple bcl2fastq analyses "
                      "being used on one run. This number should be 100%."
                  ),
+                        #This is the textbox at the bottom, hover over to see title
              )],
-            style={'width': '24%', 'display': 'inline-block'}
+            style={'width': '24%', 'display': 'inline-block', }
         ),
+                 #Pie chart, by default is set to left with a width of 24% of secreen. 
         html.Div(
             [dcc.Graph(id='unknown_index_bar')],
-            style={'width': '74%', 'display': 'inline-block', 'float': 'right'}
+            style={'width': '76%', 'display': 'inline-block', 'float': 'center'}
         ),
     ]),
     html.Div(id='sample_run_hidden', style={'display': 'none'}),
     html.Div(id='pruned_unknown_hidden', style={'display': 'none'}),
+            #html.div is just a way of dividing the HTML into meaningful groups
+            #Within the main layout division there are 4 groups: 
+            #           1. the section with the 100% bar graph
+            #           2.  Nested subdivision of: 
+            #                   a) pie graph at 24%
+            #                   b) bar graph at 76%
+            #           3. Sample Run hidden
+            #           4. Pruned  Unknown Hidden
+            #Note: span can be used for inline setting
 ])
 
 try:
-    from app import app
-except ModuleNotFoundError:
+    from app import app #From package app import app in which case the first instance of the word is the package
+except ModuleNotFoundError: #When module not found, terminate
     import dash
-    app = dash.Dash(__name__)
+    app = dash.Dash(__name__) 
     app.layout = layout
 
 
 @app.callback(
     dep.Output('sample_run_hidden', 'children'),
     [dep.Input('run_select', 'value')]
+#           Whenever input(drop down) changes, app.callback re-runs and updates     
```

Comment:
Functions should be annotated with docstring. #2 has a link to the format that should be used. A good example is

```
def function_with_types_in_docstring(param1, param2):
    """Example function with types documented in the docstring.

    `PEP 484`_ type annotations are supported. If attribute, parameter, and
    return types are annotated according to `PEP 484`_, they do not need to be
    included in the docstring:

    Args:
        param1 (int): The first parameter.
        param2 (str): The second parameter.

    Returns:
        bool: The return value. True for success, False otherwise.

"""
```

The first sentence should be a succinct explanation of what the function does. The next paragraph adds any necessary details/considerations/limitations.  The `Args` fields are used to describe the parameters that go into the function (for Dash callbacks: what causes this function to be executed) and the `Returns` field explains what the function returns (for Dash Callbacks: what does this function change).
```

Reply:
`The next paragraph adds any necessary details/considerations/limitations.` <- since Irene wasn't the one to write and debug the functions, I don't know that she has the necessary context to add these for this file. The rest should be easier to add though.